webview/src/setup/components/App.tsx

```diff
@@ -43,6 +43,8 @@ export const App: React.FC = () => {
   const [isStudioConnected, setIsStudioConnected] = useState<boolean>(false)
   const [shareLiveToStudio, setShareLiveToStudioValue] =
     useState<boolean>(false)
+  const [hasReceivedMessageFromVsCode, setHasReceivedMessageFromVsCode] =
```

Comment:
We are definitely getting close to needing to manage all of this state differently.

Reply:
IMO we're passed that and I made a comment about that a couple of PRs ago. This is obviously not in the scope of this PR, but we should plan on doing this.

Reply:
Yep, already in #3434!